src/StoreApi/Schemas/CartItemSchema.php

```diff
@@ -312,6 +324,8 @@ public function get_item_response( $cart_item ) {
 			'key'                  => $cart_item['key'],
 			'id'                   => $product->get_id(),
 			'quantity'             => wc_stock_amount( $cart_item['quantity'] ),
+			'quantity_min'         => 1,
+			'quantity_step'        => 1,
```

Comment:
What's the point of hardcoding this, how can plugins like yours access this? is it available somewhere on the cart item?

Reply:
I wasn't 100% certain if these should be part of the API, and therefore didn't add any filters there.

See

> Note that I haven't added any filters in the API schema class yet (maybe I should?)

In our initial implementation we are filtering the response via WP REST API filters -- eventually I'd love to see a way to filter certain cart line fields on the PHP side. For example, product permalinks commonly need to be amended with query arg params that restore their cart configuration when clicked in the cart (WC core already does that with variations).